wdl/gwas/regenie_step1.wdl

```diff
@@ -36,26 +36,34 @@ task step1 {
         --gz \
         --threads $n_cpu \
         --out ${prefix} \
+        --write-null-firth
         ${options}
 
         # rename loco files with phenotype names and update pred.list accordingly giving it a unique name
         awk '{orig=$2; sub(/_[0-9]+.loco.gz/, "."$1".loco.gz", $2); print "mv "orig" "$2} ' ${prefix}_pred.list | bash
         phenohash=`echo ${sep="," phenolist} | md5sum | awk '{print $1}'`
         awk '{sub(/_[0-9]+.loco.gz/, "."$1".loco.gz", $2)} 1' ${prefix}_pred.list > ${prefix}.$phenohash.pred.list
 
+        # rename firth files with phenotype names and update firth.list accordingly giving it a unique name
+        awk '{orig=$2; sub(/_[0-9]+.firth.gz/, "."$1".firth.gz", $2); print "mv "orig" "$2} ' ${prefix}_firth.list | bash
+        phenohash=`echo ${sep="," phenolist} | md5sum | awk '{print $1}'`
+        awk '{sub(/_[0-9]+.firth.gz/, "."$1".firth.gz", $2)} 1' ${prefix}_firth.list > ${prefix}.$phenohash.firth.list
+
     >>>
 
     output {
         File log = prefix + ".log"
         Array[File] loco = glob("*.loco.gz")
         File pred = glob("*.pred.list")[0]
+        Array[File] nulls = glob("*.firth.gz")
+        File firth_list = glob("*.firth.list")[0]
```

Comment:
is glob guaranteed to keep predictable order? 
You know the name of the file so you could just grab that.

Reply:
easy to modify to later to add another file ending .firth.list and this becomes unpredictable

Reply:
True, I just copypasted the loco handling as regenie handles the null firth files identically (list of pheno->file mappings and then the files)